docker-compose-local.yml

```diff
@@ -5,13 +5,10 @@ services:
     container_name: simulator-server
     environment:
       - PORT=1212
-      - KUBE_SCHEDULER_SIMULATOR_ETCD_URL=http://simulator-etcd:2379
-      - CORS_ALLOWED_ORIGIN_LIST=http://${SIMULATOR_EXTERNAL_IP:-localhost}:3000
```

Comment:
It's still needed as it's being used by simulator's server (not kube-apiserver) too.

Reply:
nvm, we don't need because simulator/config.yaml has it.